chinglee-iot/FreeRTOS-Kernel
# Flow change scanned by coverity.

Flow change scanned by coverity.

Description
-----------

Flow change scanned by coverity. Checkers:
- OVERRUN
- REVERSE_INULL

Test Steps
-----------

Scanned by coverity.

Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
